source/_components/aurora_abb_powerone.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "Aurora ABB Powerone PV Inverter Sensor"
+description: "Instructions on how to integrate an Aurora ABB Powerone solar inverter within Home Assistant."
+date: 2019-06-27 23:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```

Comment:
Please use the logo of aurora powerone

Reply:
Am I right that to add an image I need to clone then create a branch for the PR? So far this is created using the web UI.

Reply:
Done now.

Reply:
@davet2001 I've unresolved this, since the `home-assistant.png` logo is still there.

Reply:
Sorry, I committed to the wrong branch.  Should be correct now. I'm quite new to github!